pkg/webhook/certs.go

```
@@ -480,7 +482,10 @@ func (r *ReconcileVWH) Reconcile(request reconcile.Request) (reconcile.Result, e
 			return reconcile.Result{}, nil
 		}
 		log.Info("ensuring CA cert on ValidatingWebhookConfiguration")
-		injectCertToWebhook(vwh, artifacts.CertPEM)
+		if err = injectCertToWebhook(vwh, artifacts.CertPEM); err != nil {
+			log.Error(err, "unable to inject cert to webhook")
+			return reconcile.Result{}, nil
```

shouldnt we bubble up this err?

---

+1